src/stores/media/index.ts

```diff
@@ -376,5 +376,16 @@ export const useMediaStore = defineStore('media', {
         throw new Error(errorMessage)
       }
     },
+
+    setMediaProperties(
```

Comment:
Here, if the audio item was loaded on SSR, it's not in the `results` array. It's in the single audio store. I think this is the reason for the crash you get if you load an audio on SSR, and then try clicking on a related audio:
![audio](https://user-images.githubusercontent.com/15233243/174337531-c7612ac7-40da-46f0-92db-d1332411fb64.gif)

Reply:
I think we could have a helper that would return which media store this item is in: `main`, `related-media` or `media-result`. And `setMediaProperties` could then use this helper internally to update the correct store.

Reply:
Interesting.

I wonder if we should actually change the media store to be an "every piece of media ever returned by the API during this session" store and then have a separate "results" store? :thinking: